app/views/users/edit.html.slim

```diff
@@ -43,14 +43,10 @@
   = f.input :company_name, label: 'Company name'
   = f.input :company_info, label: 'Company info', hint: 'Where is the office, how much space can you provide, etc.'
 
-  = f.input :bio, label: 'About you', hint: 'You can use basic <a href="http://en.wikipedia.org/wiki/Markdown" target="_blank">Markdown</a> here.'.html_safe
```

Comment:
We still need that one. It's for the public user profile. Here's an [example](https://teams.railsgirlssummerofcode.org/users/3) (it's the 'About me' part).